### ISSUE TITLE
Error on when trying to install the lib

### ISSUE THREAD

**Issue by lrabuffetti** [antonfisher/react-simple-timefield#39]
When I try to add the library I get the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: react-simple-timefield@3.2.5
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR!   peerOptional react@"^16.8.0 || ^17.0.0 || ^18.0.0" from @apollo/client@3.7.0
npm ERR!   node_modules/@apollo/client
npm ERR!     dev @apollo/client@"^3.7.0" from the root project
npm ERR!     peer @apollo/client@"3.x" from aws-appsync-auth-link@3.0.7
npm ERR!     node_modules/aws-appsync-auth-link
npm ERR!       dev aws-appsync-auth-link@"^3.0.7" from the root project
npm ERR!   27 more (@emotion/react, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0.0 || ^17" from react-simple-timefield@3.2.5
npm ERR! node_modules/react-simple-timefield
npm ERR!   react-simple-timefield@"^3.2.5" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@"^16.0.0 || ^17" from react-simple-timefield@3.2.5
npm ERR!   node_modules/react-simple-timefield
npm ERR!     react-simple-timefield@"^3.2.5" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/circleci/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2023-01-20T18_34_43_233Z-debug-0.log


Exited with code exit status 1
```

**Comment by jasontjahjono** [comment]
@Author please update this so that we can install it in react 18

**Comment by antonfisher** [comment]
👍 should be fixed in [v3.3.1](https://github.com/antonfisher/react-simple-timefield/releases/tag/v3.3.1).
